nfsn-setup.sh

```diff
@@ -9,27 +9,24 @@ git submodule init
 git submodule update --remote
 mkdir -p letsencrypt.sh/.acme-challenges
 
+# write_config $config_dir $wellknown
+write_config() {
+    mkdir -p -- "$1" "$2"
+    printf '%s=%q\n' WELLKNOWN "$2" HOOK "$(realpath nfsn-hook.sh)" >"$1/config"
```

Comment:
Maybe should do something like `readonly hook_path="$(realpath nfsn-hook.sh)"` up at the top of the script.